Migrate Quanta/DEM documentation (#4629)
centreon/centreon-documentation: dem/installation/installation-checklist.md

---
id: installation-checklist
title: Installation checklist
---

> # DEM installation checklist

Reviewer: Here the title in the ToC is different from the title of the topic. This must be fixed for every single topic: you need to remove the title in the markdown (and update the one in the title attribute if necessary).
Also, make sure you don't have a level 1 title at the beginning of th etopic as this will replace the topic title. This is why we always start with level 2 titles.

This list covers the requirements to ensure proper operation of DEM's different modules for a given site.

# User journeys

There is **nothing to install** because DEM connects to your application just like any regular user.

> However, depending on the security level on your site, you may need to allow our IPs so that your anti-bot system does not block our probes. If that's the case, you'll find the procedure here:

Reviewer: ```suggestion
However, depending on the security level on your site, you may need to allow our IP addresses so that your anti-bot system does not block our probes. If that's the case, you'll find the procedure here:
```

[DEM IP addresses](./dem-ip-addresses.md)

# Real User Monitoring

> Real User Monitoring installs like any marketing tag: by inserting a JavaScript tag. For the full procedure and installation instructions, see:

Reviewer: The sentence sounds unfinished
"by inserting a JavaScript tag in each page's metadata" (if that's what is actually done)?

[Install Real User Monitoring](./real-user-monitoring-installation.md)

# System agent

DEM's system functionality requires at minimum the installation of system agents, then adding modules depending on the level of detail required or allowed by your license.

## Minimal installation

> 

Reviewer: both title levels are formatted the same way by Docusaurus: you lose the hierarchy between them (but it will be sorted when you remove level 1 titles)

To install the system agents, follow the detailed procedure here:

[Install system agents](./servers/install-system-agents.md)

## Installation for advanced metrics

After the agent is installed, you can install additional modules:

- Application agents for Apache, MySQL, Varnish, … to obtain information specific to those services:
    
    [Add advanced metrics](./servers/add-advanced-metrics.md)
    
- The profiler, compatible with any PHP application such as Magento or OroCommerce:
    

>     [Install PHP / Magento / OroCommerce profiler](./servers/install-php-magento-orocommerce-profiler.md)

Reviewer: ```suggestion
    [Install the PHP / Magento / OroCommerce profiler](./servers/install-php-magento-orocommerce-profiler.md)
```

    

# Automatic events

> You can add events manually in the UI but also automatically via API. **This is especially useful when you deploy a new version of your site.** We recommend setting this up so changes are recorded in DEM.

Reviewer: ```suggestion
You can add events manually in the UI, or automatically via the API. **This is especially useful when you deploy a new version of your site.** We recommend setting this up so changes are recorded in DEM.
```
